kafka-connect-s3/src/test/java/io/confluent/connect/s3/TopicPartitionWriterTest.java

```diff
@@ -513,11 +513,11 @@ public void testWriteRecordTimeBasedPartitionWallclockMockedWithScheduleRotation
     }
 
     // More records later
-    long timestampLater = time.milliseconds();
     topicPartitionWriter.write();
 
     // 11 minutes later, another scheduled rotation
-    time.sleep(11 * 60 * 1000);
+    time.sleep(TimeUnit.MINUTES.toMillis(11));
+    long timestampLater = time.milliseconds();
```

Comment:
This is the key difference in the test? That we get the `timestampLater` correctly after the `sleep`? Is this related to the fact that we seem to be combining `io.confluent.common.utils.Time` and `org.apache.kafka.common.utils.Time`?

If I'm understanding the bug here, I think this LGTM. But if it's not just the reordering, I might need more explanation...